api/src/pcapi/core/finance/models.py

```diff
@@ -719,3 +725,100 @@ class PaymentMessage(Base, Model):
     id: int = sqla.Column(sqla.BigInteger, primary_key=True, autoincrement=True)
     name: str = sqla.Column(sqla.String(50), unique=True, nullable=False)
     checksum: bytes = sqla.Column(sqla.LargeBinary(32), unique=True, nullable=False)
+
+
+##
+# When an incident is `CREATED` it can be either `CANCELLED` and it's the end of this incident. Or it can be `VALIDATED`.
+# Once `VALIDATED`, the status becomes `UNDER COMPENSATION` to trigger the functions for payments.
+# `TERMINATED` means the incident is finished and all the payments completed
+##
+class IncidentStatus(enum.Enum):
+    CREATED = "CREATED"
+    VALIDATED = "VALIDATED"
+    UNDER_COMPENSATION = "UNDER COMPENSATION"
+    TERMINATED = "TERMINATED"
+    CANCELLED = "CANCELLED"
+
+
+class IncidentType(enum.Enum):
+    OVERPAYMENT = "OVERPAYMENT"
+    COMMERCIAL_GESTURE = "COMMERCIAL GESTURE"
+    OFFER_PRICE_REGULATION = "OFFER PRICE REGULATION"
+    FRAUD = "FRAUD"
+
+
+class Incident(Base, Model):
+    id: int = sqla.Column(sqla.BigInteger, primary_key=True, autoincrement=True)
+    type: IncidentType = sqla.Column(
+        sqla.Enum(IncidentType, native_enum=False, create_contraint=False),
+        nullable=False,
+    )
+    status: IncidentStatus = sqla.Column(
+        sqla.Enum(IncidentStatus, native_enum=False, create_constraint=False),
+        nullable=False,
+        server_default=IncidentStatus.CREATED.value,
+        default=IncidentStatus.CREATED,
+    )
+
+    userId: int = sqla.Column(sqla.BigInteger, sqla.ForeignKey("user.id"), index=True, nullable=False)
+    user: sqla_orm.Mapped["users_models.User"] = sqla_orm.relationship(
+        "User", foreign_keys=[userId], backref="incidents"
+    )
+
+    # bookings: sqla_orm.Mapped[list["bookings_models.Booking"]] = sqla_orm.relationship(
+    #     "Booking", back_populates="incident"
+    # )
+    #
+    venueId = sqla.Column(sqla.BigInteger, sqla.ForeignKey("venue.id"), nullable=True)
+    venue: sqla_orm.Mapped["offerers_models.Venue | None"] = sqla_orm.relationship("Venue", foreign_keys=[venueId])
+
+    details: dict | None = sqla.Column(
+        sqla_mutable.MutableDict.as_mutable(sqla_psql.JSONB), nullable=True, default={}, server_default="{}"
```

Comment:
(détail) Je propose de rendre ce champ NOT NULL.

Si le champ est nullable, pour accéder à la valeur d'une clé dans ce champ, il faut écrire : 

```python
# soit : 
if incident.details and incident.details.get("blabla"):
    something()

# soit : 
if (incident.details or {}).get("blabla"):
    something()
```

C'est un peu lourd. Si le champ est NOT NULL, la seule valeur vide possible sera `{}` (le dictionnaire Python vide). Ce qui simplifie l'accès : 

```python
if incident.details.get("blabla"):
    something()
```